main/qemu/APKBUILD

```diff
@@ -45,7 +45,7 @@ install="$pkgname.pre-install $pkgname.post-install"
 # strip fails on .img files
 # some tests does not run on our builders
 options="suid !strip !check"
-subpackages="$pkgname-doc $pkgname-lang $pkgname-guest-agent:guest"
+subpackages="$pkgname-openrc $pkgname-doc $pkgname-lang $pkgname-guest-agent:guest"
```

Comment:
This is in conflict with package [qemu-openrc](https://pkgs.alpinelinux.org/package/edge/main/x86_64/qemu-openrc).

Reply:
Thanks, fixed!